.github/workflows/integration-test.yml

```diff
@@ -111,6 +116,7 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       version: ${{ steps.get_version.outputs.version }}
+    if: github.event_name != 'schedule'
```

Comment:
```suggestion
```
Won't this line mean there won't be a built package available to download during scheduled docker builds?

Reply:
Now that you mention this, I'm not certain what would occur. My intent was to use pre-built packages which were available during the latest push to `main`. This would largely be an effort towards reusing an unchanged package (if we have it) and reducing the need to consume resources (if we're able to).

Reply:
Reading on this more, you're correct, I'll get this updated.